src/main/java/io/mbab/sda/groupproject/entity/Song.java

```diff
@@ -0,0 +1,28 @@
+package io.mbab.sda.groupproject.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Builder
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class Song {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(length = 64, nullable = false)
+    private String author;
+
+    @Column(length = 64, nullable = false)
+    private String name;
+
+    @Column(length = 64, nullable = false)
+    private String duration;
```

Comment:
to bym trzymał w sekundach, czyi integer i pozniej odpowiednio parsowal te sekundy do wyswietlenia np. 100 s to "1:40", ale nie musisz stosowac sie do tego jak nie chcesz